Quantum-Control-Applications/NV-Centers/Confocal-Setup/time_rabi.py

```diff
@@ -46,46 +47,38 @@
 #####################################
 qmm = QuantumMachinesManager(qop_ip)
 
-qm = qmm.open_qm(config)
-
-job = qm.execute(time_rabi)  # execute QUA program
-
-res_handles = job.result_handles  # get access to handles
-counts_handle = res_handles.get("counts")
-iteration_handle = res_handles.get("iteration")
-counts_handle.wait_for_values(1)
-iteration_handle.wait_for_values(1)
-
-
-def on_close(event):
-    event.canvas.stop_event_loop()
-    job.halt()
-
-
-f = plt.figure()
-f.canvas.mpl_connect("close_event", on_close)
-next_percent = 0.1  # First time print 10%
-print("Progress =", end=" ")
-
-b_cont = res_handles.is_processing()
-b_last = not b_cont
-
-while b_cont or b_last:
-    plt.cla()
-    counts = counts_handle.fetch_all()
-    iteration = iteration_handle.fetch_all() + 1
-    if iteration / n_avg > next_percent:
-        percent = 10 * round(iteration / n_avg * 10)  # Round to nearest 10%
-        print(f"{percent}%", end=" ")
-        next_percent = percent / 100 + 0.1  # Print every 10%
-
-    plt.plot(4 * t_vec, counts / 1000 / (meas_len * 1e-9))
-    plt.xlabel("Tau [ns]")
-    plt.ylabel("Intensity [kcps]")
-    plt.title("Time Rabi")
-    plt.pause(0.1)
-
-    b_cont = res_handles.is_processing()
-    b_last = not (b_cont or b_last)
-
-print("")
+simulate = True
+if simulate:
+    simulation_config = SimulationConfig(
+        duration=28000, simulation_interface=LoopbackInterface([("con1", 3, "con1", 1)])
```

Comment:
Any specific reason for this loopback interface? Will we see something here?

Reply:
It is not really used but this way the syntax is already there in case they need it

Reply:
I see. I would remove it, but I don't have a strong opinion.

Reply:
I don't either so as a trade-off I removed it from the code except in hello_qua that can serve as a reference and that way we can show it to customers at the beginning of the session so that they are aware of it